src/nvim/ex_getln.c

```diff
@@ -1669,7 +1672,16 @@ getcmdline (
     int indent               // indent for inside conditionals
 )
 {
-  return command_line_enter(firstc, count, indent);
+  if (ui_is_external(kUICmdline)) {
+    Array args = ARRAY_DICT_INIT;
+    ui_event("cmdline_enter", args);
```

Comment:
I think it's the limitation of tui that we only draw the current state. But UI is cable to draw all the states. (they can ignore this anyway) I personally would love to see both of the commands at the same time. Isn't it the purpose of this PR to make the cmdline more presentable?